life/gender/documentation-sexism.md

```diff
@@ -0,0 +1,22 @@
+# Documentation can be sexist
+
+This is one of those things that never occurred to me, but once someone pointed
+it out I realized I'm not that surprised. (Thanks @raq929)
+
+Part of it's that I'm new to tech, so a lot of the examples I've seen were
```

Comment:
lmao that's a pretty good bad example. I think it's worth adding lol or even just screenshotting it for kicks